homeassistant/components/zha/core/helpers.py

```diff
@@ -68,7 +69,8 @@
     from zigpy.exceptions import DeliveryError
 
     attr_name = cluster.attributes.get(attr, [attr])[0]
-    attr_id = get_attr_id_by_name(cluster, attr_name)
+    attr_id = (attr, get_attr_id_by_name(cluster, attr_name))[
```

Comment:
This is a bit hard to read. Consider separating the cases on more lines.

Reply:
done